client/scripts/views/components/search_bar.ts

```diff
@@ -328,7 +327,8 @@ export const search = async (searchTerm: string, params: SearchParams, state) =>
       return token;
     });
 
-    const allComms = app.searchCache[ALL_RESULTS_KEY]['communities'];
+    const allComms = (app.config.chains.getAll() as any)
+      .concat(app.config.communities.getAll() as any);
     const filteredComms = allComms.filter((comm) => {
       return comm.name?.toLowerCase().includes(searchTerm)
```

Comment:
We should probably lowercase the search term as well.

Reply:
Also I know that this isn't part of the actual PR, but line 338 distinguishes between chains and communities by using the ... created_at field? Which seems like the wrong way to go about it.

Reply:
Re: comment one, we do convert to lowercase, before passing the searchTerm to the `search` function (e.g. ` vnode.state.searchTerm = e.target.value?.toLowerCase();`) but I'm gonna centralize that to the search function itself, so it's clearer & more easily extensible.

Re: comment two, yup, now that we're using chains/comms from config, that have been added to store already, I can just use an instanceof check, good call